src/Error/Error.php

```diff
@@ -142,11 +142,7 @@ private function updateRepr(): void
         }
 
         if ($this->name) {
-            if (\is_string($this->name) || $this->name instanceof \Stringable) {
-                $name = \sprintf('"%s"', $this->name);
-            } else {
-                $name = json_encode($this->name);
-            }
+            $name = \sprintf('"%s"', $this->name);
```

Comment:
As $name is not used anywhere else, maybe we can merge this line with the next?

Reply:
Yes, Done in https://github.com/twigphp/Twig/pull/4494/commits/8a2f2c0c1128831dd50a12b4d926f1f34ee8e693